homeassistant/components/netatmo/sensor.py

```diff
@@ -138,12 +136,13 @@ async def find_entities(data_class_name):
                 for c in data_class.get_monitored_conditions(module_id=module["_id"])
             ]
             for condition in conditions:
+                if condition not in SENSOR_TYPES:
+                    _LOGGER.debug("Unsupported condition <%s>", condition)
+                    conditions.remove(condition)
```

Comment:
You can't change a list while iterating over it.

Reply:
To say it with Raymond Hettingers words, there must be a better way.